src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -140,7 +140,7 @@ def load_arguments(self, _):
 
     with self.argument_context('backup container list') as c:
         c.argument('vault_name', vault_name_type, id_part=None)
-        c.argument('backup_management_type', backup_management_type)
+        c.argument('backup_management_type', arg_type=get_enum_type(allowed_backup_management_types + ["MAB"]), help=backup_management_type_help)
```

Comment:
Why not add `MAB` to `allowed_backup_management_types`?

Reply:
I'll check with the PM on this and get back.

Reply:
We only support selected commands for MAB(Azure Backup Agent) backup management type. So, we've only added it in the  arg_type enum wherever necessary.